content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -535,8 +535,8 @@ Administration)` businesses can **accept** or **refuse** invoices.
 
 .. seealso::
    The complete list of businesses that belong to the :abbr:`PA (Public Administration)` can be
-   found here along with their :guilabel:`Destination Code`:
-   - `List of PA businesses with their Destination Code <https://www.agenziaentrate.gov.it/portale/web/guest/aree-tematiche/fatturazione-elettronica>`_
+   found here - `List of PA businesses <https://www.agenziaentrate.gov.it/portale/web/guest/aree-tematiche/fatturazione-elettronica>`_
+   along with their Destination Code.
```

Comment:
what I meant:
  The complete list of businesses that belong to the :abbr:`PA (Public Administration)` can be
   found  `here <https://www.agenziaentrate.gov.it/portale/web/guest/aree-tematiche/fatturazione-elettronica>`_
   along with their Destination Code.